Makefile

```
@@ -28,12 +29,13 @@ GRPID   := $(shell id -g)
 .PHONY: install-hooks
 install-hooks:
 	@echo "===> Copying Antrea git hooks to local <==="
-	install hack/git_client_side_hooks/pre-commit .git/hooks/
+	@mkdir -p .git/hooks
+	install $(GIT_HOOKS) .git/hooks/
 
 .PHONY: uninstall-hooks
 uninstall-hooks:
 	@echo "===> Removing Antrea git hooks from local <==="
-	rm .git/hooks/pre-commit
+	rm -r .git/hooks
```

```suggestion
rm -r $(addprefix .git/hooks/,$(notdir $(GIT_HOOKS)))
```

See https://www.gnu.org/software/make/manual/html_node/File-Name-Functions.html